alerts/apps_alerts.libsonnet

```diff
@@ -85,7 +85,7 @@
               severity: 'critical',
             },
             annotations: {
-              message: 'StatefulSet {{ $labels.namespace }}/{{ labels.statefulset }} generation mismatch',
+              message: 'StatefulSet {{ $labels.namespace }}/{{ $labels.statefulset }} generation mismatch',
```

Comment:
I think this fix is unrelated to this PR. We should do that already in another PR (mind doing that @jolson490?).

Reply:
Sure, no problem.
Just to confirm... This PR's checks/tests (in circleci) were failing before I added that commit `8c09525` to this PR (and I assume a PR can't be merged until the checks pass?); so I'll create a separate new PR specifically to fix the existing syntax error, then once that PR is merged I'll need to update this PR branch with the latest from master (to pick up the syntax error fix)?

Reply:
Yep, that's exactly the workflow I have in mind. 👍